doc/how.rst

```diff
@@ -147,7 +147,7 @@ This ``stop_strategy`` can be:
       compute and store the objective and return ``False`` once the computations
       should stop.
 
-BenchOpt supports different types of solvers:
+Benchopt supports different types of solvers:
```

Comment:
```suggestion
benchopt supports different types of solvers:
```
?

Reply:
beginning of sentence: capitalize
else: all lower case

WDYT?

Reply:
I am +1 for @mathurinm  solution.